espymetrics/dal/sqlite_dal.py

```diff
@@ -13,25 +13,114 @@
 
 * Not human readable - Binary format makes it harder to read/grep
 * More complex - More logic = more work = more time/resources?
-
 """
 
+import os
```

Comment:
This appears to be an unused import, and while built-in, can set the wrong expectation for the following code.